types/choose_sequence/test.js

```diff
@@ -0,0 +1,29 @@
+// @todo: move this code to common stuff
+
+require("json5/lib/register");
```

Comment:
I will add to common script param to run tests jun on a single type

Reply:
https://github.com/memory-cards/card-types/pull/14

Reply:
At the PR description I've mentioned to delete this file after development will be complete.

Reply:
sure, I've just saw that for development this option might be helpfull